src/index.js

```diff
@@ -617,8 +625,26 @@ import {
 
             // Any new message that does not reply-to or mention any user (1)
             else if (eventType === ChatEventType.MESSAGE_POSTED && !targetUserId) {
+
+                /** @type {[m:(c:string) => boolean, b:(c:BotConfig, e:Election, t:string) => string][]} */
+                const rules = [
+                    [isAskedForCurrentPositions, sayNumberOfPositions]
```

Comment:
this is what I imagined the response logic would look like in the end: a list of guard/builder pairs that just gets reduced to a single message with some logging of which one matched first for debug purposes. This would allow the main module to be as clean as possible with minor tweaks being encapsulated to their own modules. 😇

Reply:
Let's do this after the SO election?

Reply:
Yeah, see my reply below - for sure any expansions must only be made after the election as there is no guarantee that current uncovered guards won't slip through. However, if you are ok with it, I'd keep it for just this one guard - I am confident in how it works, and it is covered. If there are any other rules that should match, it will simply fall-through to the rest of them